docs/resources/node_template.md

```diff
@@ -178,6 +178,7 @@ The following attributes are exported:
 * `managed_disks` - (Optional) Configures VM and availability set for managed disks. Just for Rancher v2.3.x and above. Default `false` (bool)
 * `no_public_ip` - (Optional) Do not create a public IP address for the machine. Default `false` (bool)
 * `nsg` - (Optional) Azure Network Security Group to assign this node to (accepts either a name or resource ID, default is to create a new NSG for each machine). Default `docker-machine-nsg` (string)
+* `plan` - (Optional) Azure marketplace purchase plan for Azure Virtual Machine. Format is `<publisher>:<product>:<plan>`. (string)
```

Comment:
Could you please add a comment here saying that `plan` argument is just for  Rancher v2.6.3 and above??